Cargo.toml

```diff
@@ -21,7 +21,8 @@ rust-version = "1.65.0"
 
 [workspace.dependencies]
 asn1 = { version = "0.21.1", default-features = false }
-pyo3 = { version = "0.24.2", features = ["abi3"] }
+pyo3 = { version = "0.24.2" }
```

Comment:
So we really want to always have the abi3 feature enabled, we never want to build per-python wheels. Is this change necessary?